test/decimal_test.exs

```diff
@@ -856,7 +860,7 @@ defmodule DecimalTest do
   end
 
   test "issue #60" do
-    assert_raise(ArgumentError, "cannot automatically convert nil to Decimal", fn ->
+    assert_raise(FunctionClauseError, "no function clause matching in Decimal.decimal/1", fn ->
```

Comment:
btw, way back, the assertion was `no function clause matching in Decimal.round/3`. @ericmj let me know if you'd like to get back to that so cutting private `decimal/1` conversion function from stacktrace.

(https://github.com/ericmj/decimal/pull/61/files#diff-faee63862dedd3c0321bb124e02101b1R801)

Reply:
It would be preferable but I'm not sure how to solve it without lots of boilerplate.